Homework3/Novikov_homework3.cpp

```diff
@@ -0,0 +1,261 @@
+#include <iostream>
+#include <random>
+
+void outputArray(int *array, int n) {
+  for (size_t i = 0; i < n; i++) {
+    std::cout << array[i] << " ";
+  }
+}
+
+void TaskOneReverse() {
+  // clear cin buffer after last input
+  std::cin.clear();
+  std::cin.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
+
+  constexpr int kMaxStringSize{255};
+  char string[kMaxStringSize]{};
+  std::cout << "Enter string less than " << kMaxStringSize << " symbol\n";
+
+  while (true) {
+    if (!std::cin.getline(string, kMaxStringSize)) {
+      std::cout << "Error in string range, try again\n";
+      std::cin.clear();
+      std::cin.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
+    } else if (!strlen(string))
```

Comment:
или есть другой подход чтобы не читать все строку
```suggestion
    } else if (*string != '\0')
```